### PR TITLE
Adds code of conduct

### PR DESCRIPTION
**Pull request by damacus**

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+The Short Version
```
Comment (zuazo) [inline code-of-conduct.md]: This should be a title.

File: code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Sous Chefs are dedicated to providing a harassment-free cookbook experience for everyone, regardless of gender, sexual orientation, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of Sous Chefs participants in any form.
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. 
+
+Be kind to others. Do not insult or put down other attendees. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for Sous Chef.
+
+Members violating these rules may be asked to leave the organisationat the sole discretion of [The Sous Chefs board](https://github.com/sous-chefs/meta/blob/master/membership.md).
```
Comment (zuazo) [inline code-of-conduct.md]: A space here is missing between `organisation` and `at`.

File: code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+The Short Version
+
+Sous Chefs are dedicated to providing a harassment-free cookbook experience for everyone, regardless of gender, sexual orientation, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of Sous Chefs participants in any form.
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. 
+
+Be kind to others. Do not insult or put down other attendees. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for Sous Chef.
```
Comment (zuazo) [inline code-of-conduct.md]: > put down other *community members*?

**Review by zuazo — CHANGES_REQUESTED** [review]

**Comment by damacus** [comment]
Yes I missed that. Feel free to change the PR as I can't do it right now.

**Comment by zuazo** [comment]
OK. Changes made.

Just for your information, the ChefConf CoC: https://blog.chef.io/chefconfcodeofconduct/

**Review by zuazo — APPROVED** [review]

**Comment by damacus** [comment]
Fixes #49 and #43

**Pull request merged**